documentation/ranking.html

```diff
@@ -4,27 +4,91 @@
 ---
 
 <p>
-Vespa uses <a href="reference/ranking-expressions.html">ranking expressions</a>
-to rank documents matching a query, computing a rank score per document.
-A ranking expression is stored in a <a href="#rank-profile">rank profile</a>.
-An application can have multiple rank profiles -
-this can be used for implementing different use cases, or bucket testing ranking variations.
-</p><p>
-Example: Rank the freshest document highest, using the value in an attribute named <em>timestamp</em>:
+Vespa defines Big Data Serving as:
+<blockquote style="background: #1a7db6; color: #ffffff"><strong>
+Selection, organization and machine-learned model inference,
+<ul>
+    <li>over many, constantly changing data items (thousands to billions),</li>
+    <li>with low latency (~100 ms) and high load (thousands of queries/second)</li>
```

Comment:
Not sure if we should mention 100 ms explicit, it's quite high for thousands of queries per second.

Reply:
Agree. But this phrase is used many places elsewhere in presentations / documents. Lets fine tune in next PR